specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/AppServiceCertificateOrders.json

```diff
@@ -0,0 +1,1801 @@
+          "readOnly": true
+        },
+        "rawData": {
+          "description": "Raw certificate data.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificateEmail": {
+      "description": "SSL certificate email.",
+      "type": "object",
+      "properties": {
+        "emailId": {
+          "description": "Email id.",
+          "type": "string"
+        },
+        "timeStamp": {
+          "format": "date-time",
+          "description": "Time stamp.",
+          "type": "string"
+        }
+      }
+    },
+    "CertificateOrderAction": {
+      "description": "Certificate order action.",
+      "type": "object",
+      "properties": {
+        "actionType": {
+          "description": "Action type.",
+          "enum": [
+            "CertificateIssued",
+            "CertificateOrderCanceled",
+            "CertificateOrderCreated",
+            "CertificateRevoked",
+            "DomainValidationComplete",
+            "FraudDetected",
+            "OrgNameChange",
+            "OrgValidationComplete",
+            "SanDrop",
+            "FraudCleared",
+            "CertificateExpired",
+            "CertificateExpirationWarning",
+            "FraudDocumentationRequired",
+            "Unknown"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "CertificateOrderActionType",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "Time at which the certificate action was performed.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "CertificateOrderContact": {
+      "type": "object",
+      "properties": {
+        "email": {
+          "type": "string"
+        },
+        "nameFirst": {
+          "type": "string"
+        },
+        "nameLast": {
+          "type": "string"
+        },
+        "phone": {
+          "type": "string"
+        }
+      }
+    },
+    "ReissueCertificateOrderRequest": {
+      "description": "Class representing certificate reissue request.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ReissueCertificateOrderRequest resource specific properties",
+          "type": "object",
+          "properties": {
+            "keySize": {
+              "format": "int32",
+              "description": "Certificate Key Size.",
+              "type": "integer"
+            },
+            "delayExistingRevokeInHours": {
+              "format": "int32",
+              "description": "Delay in hours to revoke existing certificate after the new certificate is issued.",
+              "type": "integer"
+            },
+            "csr": {
```

Comment:
>csr [](http://example.com/codeflow?start=13&length=3)

Generally prefer using words instead of acronyms/abbreviations (unless it's super-common like IP, PDF, etc). In any case, expand the acronym/abbreviation in the description.